.gitignore

```diff
@@ -19,3 +19,7 @@ releases/
 .*.sw[op]
 # prevent `echo foo 2>1` typo errors by making this file read-only
 1
+lang/corpus/
```

Comment:
Can we put everything inside of a fuzz/ sub directory (inside `lang/`) to keep it all self-contained? It could have its own Makefile in that dir, and .gitignore

Then we can chain the fuzz stuff in from the lang Makefile with `@$(MAKE) --quiet -C fuzz`